Webapp/sources/models/PCA_graph.py

```diff
@@ -0,0 +1,40 @@
+from sources.extensions import db
+
+from .base import Model
+
+# metadata = db.Model.metadata
+
+
+class PCAGraph(Model):
+    """
+    PCA plots can be replotted from:
+    r_class: reaction class selected in reaction class dropdown
+    colour_selected: colour in colour dropdown
+    control_points: defined points for constrained PCA
+
+    """
+
+    __tablename__ = "PCA"
+
+    id = db.Column(db.Integer, primary_key=True)
```

Comment:
Do we need date and time of creation? 
Possibly, just future work for another PR but do we want to link this to a WorkBook to enable sharing of graphs?

Reply:
yeah I included both to make it consistent with the reactions. 
Possibly, I guess we could talk to some organic chemists about specific use cases and whether they would bother collaborating using the solvent surfer or just use them personally

Reply:
Hadn't noticed in models/reactions. I don't think we ever use Reaction.date_reaction, we should remove that at some point, and I reckon this one too.
Sounds like a good idea for use cases.